app/utils/AddABeneficiaryViewHelper.scala

```diff
@@ -59,6 +59,19 @@ class AddABeneficiaryViewHelper(userAnswers: UserAnswers, draftId : String)(impl
     )
   }
 
+  private def parseCharityBeneficiary(charityBeneficiary:(CharityBeneficiaryViewModel, Int)): AddRow = {
+
+    val vm = charityBeneficiary._1
+    val index = charityBeneficiary._2
+
+    AddRow(
+      name = parseName(vm.name),
+      messages("entities.beneficiary.charity"),
```

Comment:
@JoshuaH01 part of this feature is to allow them to amend the details they have just added, this is a set to feature not available.